lib/irc_message.ml

```diff
@@ -218,7 +218,7 @@ let write_cmd_buf buf t =
   | PRIVMSG (a,b) -> pp "PRIVMSG %s %a" a write_trail b
   | NOTICE (a,b) -> pp "NOTICE %s %a" a write_trail b
   | PING a -> pp "PING %a" write_trail a
-  | PONG (a,b) -> pp "PONG %s %a" a write_trail b
+  | PONG (a,b) -> pp "PONG %a" write_trail b
```

Comment:
Should this not be

```
| PONG (a,b) -> pp "PONG %s" a
```

? Otherwise, parsing and then serialising a PONG will not give the original string, e.g.

```
# match Irc_message.parse "PONG foo.bar.baz" with
| Result.Ok message -> Irc_message.to_string message
| _ -> "error";;
- : string = "PONG "
```